test/fixtures/es-module-loaders/loader-load-foo-or-42.mjs

```diff
@@ -1,4 +1,10 @@
-export async function load(url) {
+export async function load(url, context, next) {
+  // Otherwise we can't use this loader in chaining context, as it would
+  // overwrite other subsequent loaders
```

Comment:
I don't quite understand this comment. Starting with "otherwise" makes it sound like another phrase/sentence was supposed to precede but is missing.

Reply:
Reworded it